plugins/star-rating/frontend/public/templates/widget-detail.html

```diff
@@ -0,0 +1,71 @@
+<div v-bind:class="[componentId]" class="widget-detail-view">
+	<cly-header>
+    <template v-slot:header-left>
+      <div class="widget-detail-view__widget-information">
+        <div class="widget-detail-view__back-link bu-is-size-7">
+          <i class="el-icon-back"></i> Back to Rating Widgets
+        </div>
+        <div class="widget-detail-view__widget-name bu-mt-3">
+          <h3 class="widget-detail-view__widget-name">Did you like this web page?</h3>
+        </div>
+        <div class="widget-detail-view__widget-informations bu-mt-3">
+          <div class="widget-detail-view__widget-status bu-tag"> • Running</div>
+          <div class="widget-detail-view__created-at bu-is-size-7 bu-p-1 bu-ml-2"> <i class="el-icon-time"></i> Created at 24 days ago by Lucia Deptova</div>
+          <div class="widget-detail-view__widget-id bu-is-size-7 bu-p-1 bu-ml-2"> <i class="el-icon-folder"></i> Widget ID 5fca1064e9b434744e1df82a</div>
+        </div>
+      </div>
+    </template>
+    <template v-slot:header-right>
+      <el-button class="manage-users-action-btn" size="small" icon="el-icon-video-pause">Stop Widget
+      </el-button>
+      <el-button class="manage-users-action-btn" size="small" icon="el-icon-edit">Edit Widget
+      </el-button>
+      <cly-more-options class="bu-ml-2" size="small" @command="handleCommand($event, rowScope.row._id)">
+        <el-dropdown-item command="delete-widget">Delete widget</el-dropdown-item>
+        <el-dropdown-item command="edit-widget">Edit widget</el-dropdown-item>
+        <el-dropdown-item command="show-instructions">Show instructions</el-dropdown-item>
+      </cly-more-options>
+    </template>
+  </cly-header>
+  <div class="bu-level bu-m-5 bu-has-background-white">
```

Comment:
instead of bu-m-5, i think you should wrap this entire component within cly-main, since it adds the global paddings and margins itself.

Reply:
so the idea is to wrap the main content of the view within cly-main component
Unless its headers or tabs.
header and tabs component stay outside the cly-main